.github/CHANGELOG.md

```diff
@@ -12,8 +12,34 @@
 
 ### Contributors
 
+---
```

Comment:
We shouldn't have the 0.25 dev notes here. That should only go into the main branch once we are done.